geodiff/src/geodiffcontext.cpp

```diff
@@ -20,3 +21,25 @@ const Logger &Context::logger() const
 {
   return mLogger;
 }
+
+const std::vector<std::string> &Context::tablesToSkip() const
+{
+  return mTablesToSkip;
+}
+
+void Context::setTablesToSkip( std::string &tables )
```

Comment:
I would prefer to have both the getter and setter work with std::vector<std::string>. Ideally parsing of the list of tables should be only in the CLI code.